packages/vertica-nodejs/test/integration/client/transaction-tests.js

```diff
@@ -3,9 +3,50 @@ var helper = require('./test-helper')
 const suite = new helper.Suite()
 const vertica = helper.vertica
 
+var setupTable = function (table, client) {
+  client.query("CREATE TABLE IF NOT EXISTS " + table + "(name varchar, age int);")
```

Comment:
Why not do CREATE LOCAL TEMP TABLE here?

Reply:
We have to commit and rollback